libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTrackerMixpanel.java

```diff
@@ -564,6 +564,31 @@ private AnalyticsTrackerMixpanelInstructionsForStat instructionsForStat(
                 instructions.setSuperPropertyAndPeoplePropertyToIncrement("number_of_times_accessed_stats");
                 instructions.setCurrentDateForPeopleProperty("last_time_accessed_stats");
                 break;
+            case STATS_INSIGHTS_ACCESSED:
```

Comment:
I'll make sure to copy these Mixpanel values when I do a PR for the main WPiOS repo.